qemu/tests/x86_cpu_model.py

```diff
@@ -72,6 +72,16 @@ def run(test, params, env):
     cmd = params["get_model_cmd"]
     out = session.cmd_output(cmd)
     if not re.search(guest_model, out):
+        check_cmd = params.get("check_cmd")
+        if check_cmd:
+            vulnerabilities = params.get("vulnerabilities").split(" ")
```

Comment:
The default input value of `split()` is all whitespace.

Reply:
> The default input value of `split()` is all whitespace.

Thanks and update